langchain/chat_models/azure_openai.py

```diff
@@ -110,3 +110,15 @@ def _default_params(self) -> Dict[str, Any]:
             **super()._default_params,
             "engine": self.deployment_name,
         }
+
+    def _create_chat_result(self, response: Mapping[str, Any]) -> ChatResult:
+        generations = []
+        for res in response["choices"]:
+            if "finish_reason" in res and res["finish_reason"] == "content_filter":
+                raise ValueError("Azure has not provided the response due to a content"
+                                 " filter being triggered")
+            message = _convert_dict_to_message(res["message"])
+            gen = ChatGeneration(message=message)
```

Comment:
Why we don't need to import `ChatGeneration` they need to import it for base class ChatOpenAI as well https://github.com/hwchase17/langchain/blob/master/langchain/chat_models/openai.py#L25

Reply:
Imported 👍